app/main/views/questionnaire.py

```diff
@@ -96,6 +96,19 @@ def get_summary(eq_id, form_type, collection_id):
     return redirect_to_questionnaire_page(eq_id, form_type, collection_id, latest_location)
 
 
+@questionnaire_blueprint.route('confirmation', methods=["GET"])
+@login_required
+def get_confirmation(eq_id, form_type, collection_id):
+    navigator = get_questionnaire_manager(g.schema, g.schema_json).navigator
```

Comment:
We seem to use navigator a lot but we are getting it via questionnaire_manager - should we consider a global for navigator by now?